CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 ## Unreleased changes
 
 - Expose macros for deriving `Serial` and `Deserial` from `concordium-contracts-common`.
+- Address method `is_alias_of` is now `is_alias`.
```

Comment:
Why was this changed? I think the `_of` made more sense, since you give a parameter. `is_alias` sounds more like a `() -> bool` function.

Reply:
I agree, but changing it the other way around would mean breaking changes in `concordium-contracts-common`.

Reply:
Ah, okay. Np then ;)